### PR TITLE
Bug 1136624 - [Stingray] expose isVisible and focus API for action menu

### PR DESCRIPTION
**Pull request by sean2449**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=66daf8c053f0)

File: tv_apps/smart-system/js/action_menu.js [hunk 1]
```diff
@@ -204,6 +195,34 @@
...
       }
+    },
+
+    /**
+     * Whether or not the ActionMenu is visible.
+     * @memberof ActionMenu.prototype
+     * @return {Boolean} if the container is invisible, return false
+     * @return {Number} if the container is visible, return z-index
+     */
+    isVisible: function() {
```
Comment (huchengtw-moz) [inline tv_apps/smart-system/js/action_menu.js]: Please split this method as two: isVisible return boolean and getOrder returns number.

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b7e05fef8350)

File: tv_apps/smart-system/js/action_menu.js [hunk 2]
```diff
@@ -204,6 +195,43 @@
...
+      if (this.container && this.container.classList.contains('visible')) {
```
Comment (huchengtw-moz) [inline tv_apps/smart-system/js/action_menu.js]: You may just use `return  this.container && this.container.classList.contains('visible')`

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f0a209e2b824)

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e89aefe74bdf)

**Comment by try-server-hook** [comment]
[sean2449 Sung Chiu (sean2449)](https://github.com/sean2449) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cf4862a856ae)

**Pull request merged**